index/util/docs.go

```diff
@@ -71,7 +71,24 @@ func ReadDocs(path string, n int) ([]doc.Document, error) {
 		return nil, fmt.Errorf("requested %d metrics but found %d", n, len(docs))
 	}
 
-	return docs, nil
+	return filterEmptyFields(docs), nil
+}
+
+func filterEmptyFields(docs []doc.Document) []doc.Document {
```

Comment:
Would it be worth moving this logic into `ReadDocs` so we can just remove empty fields instead of dropping the whole document? Is this a common occurrence?

Reply:
done

Reply:
done